pages/incometax.vue

```diff
@@ -0,0 +1,130 @@
+          </a>
+        </i18n>
+      </paragraph>
+      <paragraph path="incometax.procedure.p1" :margin-bottom="false"/>
+      <ul>
+        <li class="mb-1 paragraph-sm xl:paragraph"> {{ $t("incometax.procedure.options.o1") }}</li>
+        <li class="mb-1 paragraph-sm xl:paragraph">
+          <i18n path="incometax.procedure.options.o2.content" tag="p" class="paragraph-sm xl:paragraph">
+            <a
+              class="italic text-link"
+              :href="'https://static.anaf.ro/static/10/Anaf/AsistentaContribuabili_r/telefoane_judete/Regiuni.htm'"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              {{ $t("incometax.procedure.options.o2.administrations_label") }}
+            </a>
+        </i18n>
+        </li>
+        <li class="mb-1 paragraph-sm xl:paragraph"> {{ $t("incometax.procedure.options.o3") }}</li>
+        <li class="mb-1 paragraph-sm xl:paragraph"> {{ $t("incometax.procedure.options.o4") }}</li>
+      </ul>
+      <paragraph :margin-bottom="false">
+        <i18n path="incometax.procedure.gdpr.content" tag="p" class="paragraph-sm xl:paragraph">
+          <a
+          class="italic text-link"
+          :href="''"
+          target="_blank"
+          rel="noopener noreferrer"
+          >
+            {{ $t("incometax.procedure.gdpr.document_label") }}
+          </a>
+        </i18n>
+      </paragraph>
+    </div>
+    <div>
+      <section-title path="incometax.deadline.title" />
+      <paragraph :margin-bottom="false">
+        <i18n path="incometax.deadline.p1.content" tag="p" class="paragraph-sm xl:paragraph">
+          <a
+          class="italic text-link"
+          :href="'http://cdep.ro/pls/proiecte/upl_pck.proiect?idp=18123'"
+          target="_blank"
+          rel="noopener noreferrer"
+          >
+            {{ $t("incometax.deadline.p1.link_label") }}
+          </a>
+        </i18n>
+      </paragraph>
+      <paragraph path="incometax.deadline.p2" :margin-bottom="false"/>
+    </div>
+    <div>
+      <section-title path="incometax.howto.title" />
+      <paragraph path="incometax.howto.p1" :margin-bottom="false"/>
+      <ul>
+        <li
+          v-for="(value, index) in $t('incometax.howto.list')"
+          :key="index"
+          class="mb-1 paragraph-sm xl:paragraph"
+        >
+          {{ value }}
+        </li>
+      </ul>
+      <paragraph path="incometax.howto.p2" :margin-bottom="false"/>  
+    </div>
+    <faq qanda="incometax.faq.q1" />
+    <faq qanda="incometax.faq.q2" />
+    <faq qanda="incometax.faq.q3" />
+    <faq qanda="incometax.faq.q4" />
+    <faq qanda="incometax.faq.q5" />
+  </article>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'nuxt-property-decorator'
+
+import ogMetaFor from '@/lib/HeaderMeta'
+
+@Component
+export default class IncomeTax extends Vue {
+  head () {
+    return ogMetaFor({
+      route: this.$nuxt.$route.path,
+      img: '/og_supportus.jpg',
```

Comment:
We need a custom preview image here.

Reply:
@Traveler3568 do you have an image we can use here for the social preview?

Reply:
I'll get back to you by tomorrow

Reply:
@Traveler3568 here

Reply:
Choose one of these or let me know if I should make a different one. 

I suggest we use "redirect % of your income taxes" instead of "tax relief" on our public pages as the terms have different applications. My searches showed that other NGO's from Romania are also using the expression "redirect %..."

![redirectioneaza_Asociatia_Community_3](https://user-images.githubusercontent.com/69460413/105019763-24b9ba80-5a4f-11eb-8b1c-c0dbad80d310.jpg)
![redirectioneaza_Asociatia_Community](https://user-images.githubusercontent.com/69460413/105019767-25eae780-5a4f-11eb-9ae9-2a7e725f0ed0.jpg)
![redirectioneaza_Asociatia_Community_1](https://user-images.githubusercontent.com/69460413/105019768-25eae780-5a4f-11eb-8d3b-49327a266fce.jpg)
![redirectioneaza_Asociatia_Community_2](https://user-images.githubusercontent.com/69460413/105019770-26837e00-5a4f-11eb-83d3-372d7f02c6f3.jpg)

Reply:
Use the last one, personal preference, hope it's fine, if you don't like it let me know and i'll change with another one.